src/js/index.js

```diff
@@ -1,29 +1,35 @@
-require('jquery/dist/jquery');
-require('popper.js/dist/umd/popper');
-require("bootstrap/dist/js/bootstrap.bundle");
-
-
 /**
  * This file should be edited in ./src/js/index.js. After bundling the resulting file in ./pydata_sphinx_theme/static/js/index.js should never be manually changed.
  * Edit ./src/js/index.js and run yarn build:dev or yarn build:production.
  */
+import 'jquery';
+import 'popper.js';
+import 'bootstrap';
+
+import './../scss/index.scss';
```

Comment:
Is there a reason the css needs to be imported here?

Reply:
Initialy the scss was imported on Webpack level as an entry: https://github.com/pandas-dev/pydata-sphinx-theme/pull/179/files#diff-5c572bb8f24a00a84aa28f22d3a39ba5L7. Best practice in the Webpack community is to have a single javascript entry (or multiple when chunk or split bundling). 

The main advantage of only define `index.js` as the entry is that Webpack can take care of bundling all other imported assets.

Reply:
OK, I suppose I just don't really understand the relation with the actual layout.html, where we still load both index.js and index.css ?